Model/Subscriptions.cs

```diff
@@ -62,8 +62,8 @@ public class SubscriptionVerification : SubscriptionWithLease {
     }
 
     public enum SubscriptionMode {
-        Subscribe,

```

Comment:
I don't think we should change the Enum casing just to get an easy fix for `ToString()` being the correct case. Better to stick with the language convention here.